views/lib/view.js

```diff
@@ -31,10 +32,12 @@ export default class View {
     this.id = 'base_view'
     this.name = 'base view'
     this.title = 'Base View'
-    this.hasComponentData = false // True if vue supports Vue.js components
+    this.hasPrerenderedTables = false // True if vue supports Vue.js components
```

Comment:
very minor: the comment needs to be updated or removed

Reply:
updated